Editor/Window/ManagedEC2Page.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+using OperatingSystem = Amazon.GameLift.OperatingSystem;
+
+namespace AmazonGameLift.Editor
+{
+    public class ManagedEC2Page
+    {
+        private readonly VisualElement _container;
+        private readonly DeploymentSettings _model;
+        private readonly Button _deployButton;
+        private readonly Button _redeployButton;
+        private readonly Button _deleteButton;
+        private readonly Button _launchClientButton;
+        private readonly DeploymentScenariosInput _deploymentScenariosInput;
+        private readonly FleetParametersInput _fleetParamsInput;
+        private readonly Label _ec2DeploymentStatusLabel;
+
+        public ManagedEC2Page(VisualElement container)
+        {
+            _container = container;
+            _model = DeploymentSettingsFactory.Create();
+            _model.Restore();
+            _model.Refresh();
+            var parameters = new FleetParameters
+            {
+                FleetName = _model.FleetName ?? $"{Application.productName}-ManagedFleet",
+                LaunchParameters = _model.LaunchParameters ?? $"",
+                BuildName = _model.BuildName ??
+                            $"{Application.productName}-{_model.ScenarioName.Replace(" ", "_")}-Build",
+                GameServerFile = _model.BuildFilePath,
+                GameServerFolder = _model.BuildFolderPath,
+                OperatingSystem = FleetParametersInput.GetOperatingSystem(_model.BuildOperatingSystem) ??
+                                  OperatingSystem.AMAZON_LINUX_2
+            };
+
+            var mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/Pages/ManagedEC2Page");
+            var uxml = mVisualTreeAsset.Instantiate();
+
+            container.Add(uxml);
+
+            var ec2Deployment = new ManagedEC2Deployment(_model, parameters);
+            var scenarioContainer = container.Q("ManagedEC2ScenarioTitle");
```

Comment:
Is this correct? You are using the element that is meant for the title text as the container itself:
```
<ui:Label name="ManagedEC2Title" class="page__title" text="Managed EC2"/>
```

Should you be using it's parent instead?

Reply:
It is using the parent in `ManagedEC2Page.uxml`
```xml
<ui:Foldout name="ManagedEC2ScenarioTitle" class="foldout" text="Select Deployment Scenario"/>
```